### ISSUE TITLE
Why block huggingface.co?

### ISSUE THREAD

**Issue by havardox** [laylavish/uBlockOrigin-HUGE-AI-Blocklist#40]
I'm a programmer and this site is defintely not spam. It's one of the largest sources of machine learning models out there. Akin to blocking stackoverflow.com if you program in ML

**Comment by laylavish** [comment]
Huggingface is an AI-centric platform that hosts Stable Diffusion checkpoints and loras, quantized llm models, and other ML models. They also host AI generated imagery on there, of which appears in image search engines. This is a blacklist meant to remove AI generated content from searches. Why wouldn't I put huggingface on this list? 

This is a list, and it's customizable. If you want to clear huggingface, there's no harm in just whitelisting it.

**Comment by havardox** [comment]
I guess I don't exactly understand why huggingface.co hosting AI generated imaginary would warrant blocking it entirely. I understand blocking SEO blogspam and the likes as it would be _very_ unlikely to be of interest to a person, but huggingface.co still hosts a ton of useful content, including machine learning models, datasets, library documentation, ML guides (that a person might use to learn) and other stuff

A person can just whitelist and move on with their merry way; that's true. It's just a little unexpected. I myself was a bit befuddled as to why I couldn't get any huggingface results on Google until I realized this was included in the blacklist I just installed
